src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/PerformanceElementTiming.kt

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.performance.tools.jiraactions.api.w3c
```

Comment:
Let's add the new API to the CHANGELOG.md

Reply:
- [] Do it!